ui/main/src/app/services/acknowlegment/AcknowledgeService.ts

```
@@ -30,140 +20,17 @@ export class AcknowledgeService {
     }
 
-    public static postUserAcknowledgement(cardUid: string, entitiesAcks: string[]): Observable<ServerResponse<void>> {
-        return AcknowledgeService.acknowledgeServer.postUserAcknowledgement(cardUid, entitiesAcks);
+    public static deleteAcknowledgement(cardUid: string) {
+        return AcknowledgeService.acknowledgeServer.deleteUserAcknowledgement(cardUid, this.getAcknowledgedEntities());
     }
 
-    public static deleteUserAcknowledgement(cardUid: string, entitiesAcks: string[]): Observable<ServerResponse<void>> {
-        return AcknowledgeService.acknowledgeServer.deleteUserAcknowledgement(cardUid, entitiesAcks);
+    public static postAcknowledgement(cardUid: string): Observable<ServerResponse<void>> {
+        return AcknowledgeService.acknowledgeServer.postUserAcknowledgement(cardUid, this.getAcknowledgedEntities());
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)